packages/react-data-grid/src/__tests__/utils/index.ts

```diff
@@ -1,18 +1,20 @@
+import { valueCellContentRenderer } from '../../Cell/cellContentRenderers';
 import { CalculatedColumn } from '../../common/types';
 
-function createColumn(index: number): CalculatedColumn<{ [key: string]: unknown }> {
+function createColumn(index: number): CalculatedColumn<{ [key: string]: React.ReactNode }> {
```

Comment:
Is this correct? [This line](https://github.com/adazzle/react-data-grid/blob/canary/packages/react-data-grid/src/common/types.ts#L50) stands for the RowData type, doesn't it? Then probably we should not return the `ReactNode`?

Reply:
`valueCellContentRenderer` needs to return a property on `RowData`, and cell content formatters, need to return a `React.ReactNode`, so that's why I made the change here.
In tests we generally just use string or number values, I'm just making the type broad so it's easier to deal with.

Reply:
I believe `React.ReactNode` is too broad which is similar to `any` but missing some basic types and adding some unnecessary types, e.g. a field in the row data could have an object or an array of objects which is not defined in `React.ReactNode`. In addition in terms of considering what is valid data type, I believe it's inappropriate as well because generally speaking, a `Row` data object should not contain `fragment` for example. (Though with that being said, people could still potentially save an element in the row data, but generally speaking they won't save into database) My point was that for RDG code, it actually doesn't care about what does the `RowData` looks like, so I think `CalculatedColumn<{ [key: string]: any }>` would be more appropriate in this case if `CalculatedColumn<{ [key: string]: unknown }>` will cause test type issue.